modules/acquia_lift_publisher/acquia_lift_publisher.module

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * @file
+ * Drupal Module: Acquia Lift - Publisher.
+ *
+ * Acquia Content Hub Publisher exports content from your Drupal site to
+ * Content Hub services. This module alters the behavior of the default
+ * functionality.
+ */
+
+use Drupal\acquia_lift_publisher\Form\AcquiaLiftPublisherSettingsForm;
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\Core\Entity\EntityInterface;
+
+/**
+ * Implements hook_entity_insert().
+ */
+function acquia_lift_publisher_entity_insert(EntityInterface $entity) {
+  _acquia_lift_publisher_trigger_queue($entity);
+}
+
+/**
+ * Implements hook_entity_update().
+ */
+function acquia_lift_publisher_entity_update(EntityInterface $entity) {
+  _acquia_lift_publisher_trigger_queue($entity);
+}
+
+/**
+ * Triggers the Content Hub export process.
+ *
+ * @param \Drupal\Core\Entity\EntityInterface $entity
+ *   The current entity.
+ */
+function _acquia_lift_publisher_trigger_queue(EntityInterface $entity): void {
```

Comment:
This function is better suited in a separate class file in the `src/EventSubscriber/Publish` directory. `acquia_lift` modules attempts to keep the `.module` files as small as possible.

Reply:
Moved to separate class.